### DISCUSSION TITLE
Cannot Generate Repository from Organization Template

### DISCUSSION THREAD

**Discussion by edgarmartinez-43** [community/community#41028] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Hello, I'm running into issues with the API when creating a new repository from a template

My implementation uses the given github provider in terraform ([Repository here](https://github.com/integrations/terraform-provider-github)). However, upon applying changes in terraform I see this API response:
```
POST https://api.github.com/repos/my-org/my-template-repo/generate: 404 Not Found []
```
I had assumed I've had something misconfigured on my end so I did a curl request with the corresponding documentation listed [here](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-using-a-template) and got the resulting response when properly authenticated:
```
curl \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer $GITHUB_TOKEN"\
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/my-org/my-template-repo/generate
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}
```
It could be the Documentation is outdated or this endpoint has been disabled recently

**Comment by ossanna16** [comment]
Hi there @edgarmartinez-43 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
